parser-wrapper/raml-parser-interface/src/main/java/org/mule/raml/interfaces/model/IQueryString.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.raml.interfaces.model;
+
+import org.mule.raml.interfaces.model.parameter.IParameter;
+
+import java.util.Map;
+
+public interface IQueryString {
+
+  String getDefaultValue();
+
+  boolean isArray();
+
+  boolean validate(String value);
+
+  boolean isScalar();
+
+  boolean isFacetArray(String facet);
+
+  Map<String, IParameter> facetsWithDefault();
```

Comment:
I would change this method by just returning all parameters being the consumers who filter that list then